certified-connectors/tyntec-Viber/apiDefinition.swagger.json

```diff
@@ -343,6 +363,594 @@
+        "x-ms-trigger": "single",
+        "deprecated": false,
+        "x-ms-api-annotation": {
+          "family": "Incoming",
+          "revision": 2
+        },
+        "parameters": [
+          {
+            "name": "viberServiceId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Your Viber Service ID",
+            "x-ms-summary": "Your Viber Service ID"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "inboundMessageUrl": {
+                  "type": "string",
+                  "description": "inboundMessageUrl",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": ""
+                }
+              },
+              "required": [
+                "inboundMessageUrl"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/conversations/v3/messages/{messageId}/status": {
+      "get": {
+        "responses": {
+          "202": {
+            "description": "Returned when the operation successfully connects to Conversations API",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "messageId": {
+                  "type": "string",
+                  "description": "messageId",
+                  "x-ms-visibility": "important"
+                },
+                "deliveryChannel": {
+                  "type": "string",
+                  "description": "deliveryChannel",
+                  "x-ms-visibility": "important"
+                },
+                "status": {
+                  "type": "string",
+                  "description": "status",
+                  "x-ms-visibility": "important"
+                },
+                "timestamp": {
+                  "type": "string",
+                  "description": "timestamp",
+                  "x-ms-visibility": "important"
+                }
+              }
+            }
+          }
+        },
+        "operationId": "StatusCheckV3",
+        "summary": "Status Check",
+        "description": "Check for status of your message",
+        "deprecated": false,
+        "x-ms-api-annotation": {
+          "family": "GetStatus",
+          "revision": 2
+        },
+        "parameters": [
+          {
+            "name": "messageId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Message ID of your message",
+            "x-ms-summary": "Message ID of your message"
+          }
+        ]
+      }
+    },
+    "/conversations/v3/configurations/channels/viber": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@ridlees empty schema is not allowed. please define a proper schema.

Reply:
Added a custom schema